optuna/multi_objective/_hypervolume/base.py

```diff
@@ -4,11 +4,40 @@
 
 
 class BaseHypervolume(object, metaclass=abc.ABCMeta):
-    """Base class for hypervolume calculators"""
+    """Base class for hypervolume calculators.
+
+        .. note::
+            This class is used for computing the hypervolumes of points in multi-objective space.
+            Each coordinate of each point represents one value of the multi-objective function.
+
+        .. note::
+            We check that each objective is to be minimized. Transform objective values that are
+            to be maximized before calling this class's `compute` method. For example,
+
+            .. testcode::
+                def objective(trial):
+                    ...
+                    return accuracy, elapsed_time
+
+                study = optuna.multi_objective.create_study(["maximize", "minimize"])
+                study.optimize(objective, n_trials=100)
+                trials = study.get_pareto_front_trials()
+                solution_sets = np.ndarray([t.values for t in trials])
+                # Transform the objective to be maximized into that for minimization.
+                solution_sets = np.ndarray([[-s[0], s[1]] for s in solution_sets])
+                reference_point = 2 * np.max(solution_set, axis=0)
+                hypervolume = WFG().compute(solution_sets, reference_point)
+                print("Hypervolume of the Pareto solutions is {}.".format(hypervolume))
```

Comment:
Hm, does the testcode run? There are undefined variables and you're using the `Ellipsis` (`...`) in what seems to be an unintended way. 🤔

Reply:
Thanks for the comment! I define the objective without any ellipsis. PTAL.